src/Generator.php

```diff
@@ -119,7 +119,7 @@ public function __construct(array $settings = null)
 	 */
 	public function setSecure($secure = true)
 	{
-		$this->secure = (bool) $secure;
+		$this->secure = (bool)$secure;
```

Comment:
It's not a PSR standard, so technically it's down to developer preference. But my argument would be that a space between the cast and the variable is more consistent with other PSR standards.